### ISSUE TITLE
Question about the calculation of  the unit cost for cement production.

### ISSUE THREAD

**Issue by RawisWar** [JGCRI/gcam-core#114]
Hi 
In reference to the [posts](https://github.com/JGCRI/gcam-core/issues/99) by @robbieorvis and @pkyle , I understand the energy and raw material inputs for the cement sector in GCAM are modeled as Leontief Inputs.  I need help understanding how GCAM computes the  unit cost of cement production (in 1975$/kg). After reviewing the [minicam_leontief_production_function](https://github.com/JGCRI/gcam-core/blob/master/cvs/objects/functions/source/minicam_leontief_production_function.cpp)  and [leontief_production_function](https://github.com/JGCRI/gcam-core/blob/master/cvs/objects/functions/source/leontief_production_function.cpp) functions, I am still unclear about how the cost for cement is generated.  Any help in explaining how GCAM  handles the cost of cement will be greatly appreciated. 

Thanks

**Comment by pkyle** [comment]
The price of cement in the model in the base year is from $0.015 to $0.026, in 1975$/kg. In 2010 dollars, this is about $48 to $85 per tonne. The specific prices aren't something we've focused on in our model development since this module was added in about 2009. That said just from an internet search, present-day prices in current dollars are around $100/tonne.

The way the price is calculated is the sum of the "non-fuel" cost of cement production, assumed to be $0.005, and the prices of each of the 3 inputs (`limestone`, `process heat cement`, `elect_td_ind`) multiplied by their respective input-output coefficients. The limestone coefficient is generally about 1.5 and the price is $0.001; this is mostly tracked for emissions purposes, but does contribute to the cost. The others depend on the energy intensity of cement production in each region, and the energy prices in each region. The coefficient of `process heat cement` is about 0.004 GJ per kg (or about 4 GJ per tonne), and the price is the weighted average of the fuel prices used to produce it (generally about $2-4/GJ, in 1975$), plus a non-fuel cost of $1/GJ. The electricity coefficient is about 0.0005 (0.5GJ/t) in each region, and prices are about $10/GJ. So, each of the three inputs contributes to the prices seen in the model.
